public/content/docs/mels-hack/index.en.md

```diff
@@ -0,0 +1,173 @@
+
+1. A test, like comparing two numbers, followed immediately by...
+2. ...The `TBC` instruction, which would transfer control to the instruction in address `(A)` if the test was successful
+3. If the test failed (`else`), the program would proceed as usual to the next address in field `(N)`. 
+
+It's reasonable to assume that standard training on the RPC-4000 included only this variant of the `TBC` usage, being an essential part of computer programming. Thus, it's also reasonable to assume that Ed Nather was surprised to find a `TBC` instruction without the necessary preceding test.
+
+Instead of running a test, Mel kept incrementing the value of the `(A)` field, as described in the story. This eventually led to an overflow of the entire register, provided the index register bit `(X)` was on - exactly as Nather remembered. Using `101` as the `TBC` opcode yields the following sequence:
+
+             1111111101             
+        MSB <----------> LSB                                
+
+        +    0000001000  
+        ====================
+             0000000101
+        MSB <----------> LSB 
+
+        +     OVERFLOW
+
+The overflow would toggle the `BCU` on, causing the heretofore ineffective `TBC` to transfer control to the address in the (A) field, which was `0`. If Ed Nather was not familiar with the overflow aspect of the `BCU`, then his reading of the code would indeed lead to the diagnosis of a loop without a test. A standard conditional jump consisted of some test, followed by a `TBC` instruction, which would `JUMP` out of the loop only if the test had succeeded. It was quite natural, then, for a RPC-4000 programmer to come across a "free floating" `TBC` instruction, with no preceding test and conclude:
+
+        But the loop had no test in it.
+
+This scenario seems closer to the original story: The `JUMP` is there, as well as the overflow and the seemingly unnecessary `1` in the index register bit. There are only two deviations from the original account:
+
+1. The opcode is never modified.
+2. The magical nature of the hack is due not only to Mel's prowess, but also - perhaps mostly - to Ed Nather's incomplete understanding of the machine. Had he known that the `TBC` instruction was influenced by an overflow, he would have cracked the problem right away, leaving no story for posterity. 
+
+Anyone who likes coding can imagine Mel Kaye's reaction, when he read about the path of an overflow triggering a jump. The challenge was immediately obvious. Having run a trivial POC, he naturally proceeded to the next level, causing the overflow at the end of an iterative process, after the data had been drained. This way, he could avoid the standard test at the end of the loop. The stroke of genius was completed when he managed to usefully incorporate this structure into a real-world program. 
+
+### So What You're Saying is...
```

Comment:
Ellipsis should be removed; however - I strongly suggest to replace it with a clearer title.

Reply:
This expression became sort of an idiom a few years ago. Evidently not in your circles. I don't mind that it rings off to most readers, as long as the few that know it understand the irony.